extensions/google-chrome/src/new-tab.tsx

```diff
@@ -33,18 +34,33 @@ export default function Command() {
     return errorViewTab || errorViewHistory;
   }
 
+  const classifiedInput: ClassifiedInput = classifyInput(searchText || "");
+  let actionTitle: string;
+  let actions: ReactNode;
+  let icon: Icon;
+
+  if (!classifiedInput.value) {
+    actionTitle = "Open Empty Tab";
+    icon = Icon.Plus;
+    actions = <ChromeActions.NewTab />;
+  } else if (classifiedInput.type === "url") {
+    actionTitle = `Open URL "${searchText}"`;
+    icon = Icon.Globe;
+    actions = <ChromeActions.NewTab url={classifiedInput.value} />;
```

Comment:
greptileai said:
> * Fixed inconsistency between `searchText` and `classifiedInput.value` usage in URL handling

but I did not do it!

I think it makes more sense to reflect what is actually typed in the actionTitle instead of the extended url with "https://www" prepended.